src/comps/Header.tsx

```diff
@@ -102,11 +102,6 @@ const Header = ({ config }: { config: BridgeConfig }) => {
             {/* <h5 className="font-Matter text-xs text-black tracking-wide ">
               LEARN MORE
             </h5> */}
-            {isConnected && config.FEATURE_FLAGS.includes("withdrawals") && (
```

Comment:
since now withdrawal UI is no longer feature gated, can we also remove the withdrawal redirect gate?

Reply:
makesh shense

Reply:
done